src/YPopupInternal.h

```diff
@@ -0,0 +1,25 @@
+
+#include <string>
+#include <vector>
+
+// Internal helper class
```

Comment:
This description is too generic. We should say that it is considered a helper for YDialogSpy, perhaps even move it under its namespace.